### DISCUSSION TITLE
How to change colour of parameter when device fails

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#4348] [Community Q&A] [answered]
Suppose we are taking telemetry from three devices in open MCT. If one of the device fails, is there any option in open MCT to change colour and beep an alarm if such event occurs.

**Comment by akhenry** [comment, marked as answer]
It depends on how device failure is detected. If it is sufficient to trigger an alarm if telemetry is not received within a certain amount of time (eg. 30s) then you could configure a Condition Set using the "any data received" criterion to emit a value when the telemetry goes stale. You could then use conditional styling to highlight plots, tables, and values when telemetry goes stale. You could also create a Condition Widget to use as a visual alarm in your layouts.

For more details on condition sets, styling, and widgets, take a look at our [User's Guide](file:///Users/akhenry/Downloads/Open_MCT_Users_Guide-1.pdf).
